samples/src/Microsoft.Azure.IIoT.App/src/wwwroot/css/site.css

```diff
@@ -103,6 +103,33 @@ app {
     padding-top: 1.1rem;
 }
 
+table {
+    width: 100%;
+    table-layout: fixed;
+}
+
+td {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+td:hover {
+    overflow: visible;
+    position: relative;
+    display: inline-block;
```

Comment:
to fix the issue of the table border you can try to use z-index in td:hover
setting a z-index higher than the normal and remove the inline-block

Reply:
It didn't work. I'll open a new bug for this